examples/clipping_planes_interactive.py

```diff
@@ -0,0 +1,127 @@
+        event.dims_displayed,
+    )
+
+    # Calculate intersection of click with plane through data
+    intersection = volume_layer.experimental_clipping_planes[0].intersect_with_line(
+        line_position=near_point, line_direction=event.view_direction
+    )
+
+    # Check if click was on plane by checking if intersection occurs within
+    # data bounding box. If so, exit early.
+    if not point_in_bounding_box(intersection, volume_layer.extent.data):
+        return
+
+    # Get plane parameters in vispy coordinates (zyx -> xyz)
+    plane_normal_data_vispy = np.array(volume_layer.experimental_clipping_planes[0].normal)[[2, 1, 0]]
+    plane_position_data_vispy = np.array(volume_layer.experimental_clipping_planes[0].position)[[2, 1, 0]]
+
+    # Get transform which maps from data (vispy) to canvas
+    visual2canvas = viewer.window.qt_viewer.layer_to_visual[volume_layer].node.get_transform(
+        map_from="visual", map_to="canvas"
+    )
+
+    # Find start and end positions of plane normal in canvas coordinates
+    plane_normal_start_canvas = visual2canvas.map(plane_position_data_vispy)
+    plane_normal_end_canvas = visual2canvas.map(plane_position_data_vispy + plane_normal_data_vispy)
+
+    # Calculate plane normal vector in canvas coordinates
+    plane_normal_canv = (plane_normal_end_canvas - plane_normal_start_canvas)[[0, 1]]
+    plane_normal_canv_normalised = (
+            plane_normal_canv / np.linalg.norm(plane_normal_canv)
+    )
+
+    # Disable interactivity during plane drag
+    volume_layer.interactive = False
+
+    # Store original plane position and start position in canvas coordinates
+    original_plane_position = volume_layer.experimental_clipping_planes[0].position
+    start_position_canv = event.pos
+
+    yield
+    while event.type == "mouse_move":
+        # Get end position in canvas coordinates
+        end_position_canv = event.pos
+
+        # Calculate drag vector in canvas coordinates
+        drag_vector_canv = end_position_canv - start_position_canv
+
+        # Project the drag vector onto the plane normal vector
+        # (in canvas coorinates)
+        drag_projection_on_plane_normal = np.dot(
+            drag_vector_canv, plane_normal_canv_normalised
+        )
+
+        # Update position of plane according to drag vector
+        # only update if plane position is within data bounding box
+        drag_distance_data = drag_projection_on_plane_normal / np.linalg.norm(plane_normal_canv)
+        updated_position = original_plane_position + drag_distance_data * np.array(
+            volume_layer.experimental_clipping_planes[0].normal)
+
+        if point_in_bounding_box(updated_position, volume_layer.extent.data):
+            volume_layer.experimental_clipping_planes[0].position = updated_position
+
+        yield
+
+    # Re-enable
+    volume_layer.interactive = True
+
+
+viewer.axes.visible = True
+viewer.camera.angles = (45, 45, 45)
+viewer.camera.zoom = 5
```

Comment:
for these interactive examples I've moved to adding a small explanation in the [`Viewer.text_overlay`](https://github.com/napari/napari/blob/master/examples/viewer_fps_label.py) - could do the same here but not strictly necessary!

Reply:
I have tried to set a value and set it to visible, but nothing appears. Am I missing something?
```python
viewer.text_overlay.text = 'test'
viewer.text_overlay.visible = True
```

Reply:
hmm, I always do this in the other order (visible then change the text), might be a bug?

Reply:
Yup, seems like it!

Reply:
Actually, I can't reproduce... I must have messed up something?